### PR TITLE
Added kc to employees list

### PR DESCRIPTION
**Pull request by krishnaglick**
Fighting with the image

### CODE REVIEW

File: components/home/whoWeAre.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Row, Col } from "react-bootstrap";
+import { Col, Row } from "react-bootstrap";
```
Comment (krishnaglick) [inline components/home/whoWeAre.tsx:1]: Prettier

File: components/home/whoWeAre.tsx [hunk 2]
```diff
@@ -196,6 +197,32 @@ export default function WhoWeAre() {
...
+                    target="_blank"
+                    rel="noreferrer nofollow"
+                  >
+                    @iHeartMedia
+                  </a>
+                </span>
+              }
+              github="https://github.com/krishnaglick"
+              linkedIn="https://twitter.com/msdatadetective"
+              twitter=""
```
Comment (krishnaglick) [inline components/home/whoWeAre.tsx]: Should I make a twitter for myself as a grouparoo employee?

Comment (evantahler) [reply components/home/whoWeAre.tsx]: If you don't have a Twitter account, no need to make a new one.  In that case, the component should be updated to handle a "" or null twitter account to hide the icon

File: components/home/whoWeAre.tsx [hunk 3]
```diff
@@ -196,6 +197,32 @@ export default function WhoWeAre() {
...
+                    href="https://www.iheart.com"
+                    target="_blank"
+                    rel="noreferrer nofollow"
+                  >
+                    @iHeartMedia
+                  </a>
+                </span>
+              }
+              github="https://github.com/krishnaglick"
+              linkedIn="https://twitter.com/msdatadetective"
```
Comment (evantahler) [inline components/home/whoWeAre.tsx]: I don't think that's you...

**Review by evantahler — CHANGES_REQUESTED** [review]

File: components/home/whoWeAre.tsx (hunk 3 above)
Comment (krishnaglick) [reply components/home/whoWeAre.tsx]: Fixed!

**Pull request closed without merging**